docs/writing_plugins.rst

```diff
@@ -8,10 +8,11 @@ Helper Functions
 
 Plugins are comprised of "helper" functions that augment the behavior of `apispec.APISpec` methods.
 
-There are three types of helper functions:
+There are four types of helper functions:
 
 * Definition helpers
 * Path helpers
+* Operation resolvers
```

Comment:
@yoichi Would it make sense to call these "Operation helpers" for consistency?

Reply:
Yes. They don't have to resolve something and call them helpers like existing ones seems better. I'll push a change.
Thanks for the suggestion